src/cloud/ops.rs

```diff
@@ -85,19 +98,49 @@ pub struct Version {
     pub version: String,
 }
 
+#[derive(Debug, serde::Deserialize)]
```

Comment:
hm..., weird? why is github including this in the diff?

Reply:
no clue